src/sipnet/sipnet.c

```diff
@@ -807,6 +498,9 @@ void calcLightEff(double *lightEff, double lai, double par) {
   // Information on the distribution of LAI with height is available
   // as of March 2007 ... contact Dr. Maggie Prater Maggie.Prater@colorado.edu
 
+  // The bulk of this modeling comes from [1], except for the fAPAR tracking
```

Comment:
fAPAR comes from:

Zobitz, J.M., Moore, D.J.P., Quaife, T., Braswell, B.H., Bergeson, A., Anthony, J.A., Monson, R.K., 2014. Joint data assimilation of satellite reflectance and net ecosystem exchange data constrains ecosystem carbon fluxes at a high-elevation subalpine forest. Agric. For. Meteorol. 195–196, 73–88. https://doi.org/10.1016/j.agrformet.2014.04.011